VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/start-private-consultation/start-private-consultation.component.scss

```diff
@@ -66,4 +66,26 @@
 .vh-scroll-wrap {
   max-height: 350px;
   overflow-y: auto;
+  overflow-x: hidden;
+}
+
+@media only screen and (max-height: 736px) {
+  .vh-scroll-wrap {
+    max-height: 250px;
+    overflow-y: auto;
+    overflow-x: hidden;
+  }
+}
+
+::-webkit-scrollbar {
+  -webkit-appearance: none;
+  width: 10px;
+  overflow-x: hidden;
+  background-color: darkgrey;
```

Comment:
this is a different dark grey to the gov one or the thumb wouldnt show i guess?

Reply:
what does -webkit-appearance: none; do?

Reply:
appearance: none is to remove platform-specific styling to an element that does have it by default. As I need to make scrolling visible on mac chrome ( it's visible only by hover). dark grey is a different colour to govuk dark-grey, I could not find suitable on govuk